hits_to_page/mapper.py

```diff
@@ -0,0 +1,10 @@
+import sys
+
+"""Display the number of hits for each different file on Web site"""
+
+for line in sys.stdin:
+
+    data = line.strip().split(' ')
+
+    print("{0}\t{1}".format(data[6], data[0]))
```

Comment:
`data[6], data[0]` isn't really descriptive.
I would suggest to use unpacking: `ip, identity, username, datetime, tz, method, page, protocol, status, content_size = data`
By the way, you don't need to pass `ip` to reducer at all. All the information you'll need is present in `page`.